Makefile

```diff
@@ -30,7 +30,7 @@ setup_dev_db: ## Set up the development db on a local postgres
 
 .PHONY: reset_dev_db
 reset_dev_db: ## Reset the dev db
-	dropdb consultations_dev
+	-dropdb consultations_dev
```

Comment:
This `-` discards the exit code, so it doesn't bail on first setup if the DB doesn't exist yet